docker-compose.yml

```diff
@@ -37,20 +37,20 @@ services:
     networks:
       - primenet
 ###################################################### Web Frontend #
-  nginx-angular:
-    container_name: primeweb
-    build:
-        context: prime-angular-frontend/
-    command: "nginx -g 'daemon off;'"
-    restart: always
-    ports:
-      - "8080:8080"
-      - "4200:8080"
-    expose:
-      - "8080"
-      - "4200"
-    networks:
-      - primenet
+  # nginx-angular:
```

Comment:
I'm going to assume this was accidentally committed :P

Reply:
oops yup, let me remove that